chia/_tests/clvm/test_curry_and_treehash.py

```diff
@@ -21,3 +32,24 @@ def test_curry_and_treehash() -> None:
         hashed_args = [Program.to(_).get_tree_hash() for _ in args]
         puzzle_hash_via_f = curry_and_treehash(quoted_mod_hash, *hashed_args)
         assert puzzle_hash_via_curry == puzzle_hash_via_f
+
+
+@pytest.mark.parametrize("value", [[], [bytes32([3] * 32)], [bytes32([0] * 32), bytes32([1] * 32)]])
+def test_shatree_byte_list(value: List[bytes32]) -> None:
```

Comment:
```suggestion
def test_shatree_byte_list(value: List[bytes32]) -> None:
def test_shatree_byte_list(value: List[bytes]) -> None:
```
and maybe include some `bytes` objects of other sizes.